mongo/database.go

```diff
@@ -982,3 +967,62 @@ func (db *Database) executeCreateOperation(ctx context.Context, op *operation.Cr
 
 	return replaceErrors(op.Execute(ctx))
 }
+
+// GridFSBucket is used to construct a GridFS bucket which can be used as a
+// container for files.
+func (db *Database) GridFSBucket(opts ...*options.BucketOptions) (*GridFSBucket, error) {
+	b := &GridFSBucket{
+		name:      "fs",
+		chunkSize: DefaultGridFSChunkSize,
+		db:        db,
+		wc:        db.writeConcern,
+		rc:        db.readConcern,
+		rp:        db.readPreference,
```

Comment:
Optional: Are these values actually necessary? Is it possible to let the `Collection` inherit the values from the `Database` and only override them if different options are specified in `BucketOptions`?

Reply:
Yeah that sounds reasonable